docs/beyond_basics/101-147-riastores.rst

```diff
@@ -0,0 +1,704 @@
+.. _riastore:
+
+Remote Indexed Archives for dataset storage and backup
+------------------------------------------------------
+
+If DataLad datasets should be backed-up, made available for collaborations
+with others, or stored or managed in a central location,
+:term:`remote indexed archive (RIA) store`\s, dataset storage
+locations that allow for access to and collaboration on DataLad datasets, may be
+a suitable solution. They are flat, flexible, file-system based repository
+representations of any number of datasets, and they can exist on all standard computing
+infrastructure, be it personal computers, servers or compute clusters, or even
+super computing infrastructure -- even on machines that do not have DataLad
+installed.
+
+.. note::
+
+   Setting up and interacting with RIA stores requires DataLad version ``0.13.0``
+   or higher. In order to understand this section, some knowledge on Git-internals
+   and overcoming any fear of how checksums and UUIDs look can be helpful.
+
+Technical details
+^^^^^^^^^^^^^^^^^
+
+RIA stores can be created or extended with a single command inside of any
```

Comment:
"from within" instead of "inside of"?
It sounds a bit as if the store would be created inside the dataset.

Reply:
hihi, true